### ISSUE TITLE
Show confirmation dialog before opening many Get Info dialogs

### ISSUE THREAD

**Issue by joshas** [fman-community/fman#210]
On macOs it is easy to accidentally open a lot of Get Info dialogs having selected multiple files and expecting the same behavior as on other OS (single file properties dialog window opened).
Would be nice if confirmation dialog warned user about potential mess on his desktop. And add "never notify again" for more experienced users.

**Comment by gingerbeardman** [comment]
I would ask for behaviour to follow macOS rather than add in a confirmation dialog?

macOS displays Get Info as follows:

<kbd>Cmd</kbd>+<kbd>i</kbd> = single item info, one window per file/folder (currently supported in fman)
![screen shot 2018-07-31 at 09 53 39](https://user-images.githubusercontent.com/49612/43449152-a5bd8c76-94a7-11e8-81e1-21836ce1295c.png)

<kbd>Cmd</kbd>+<kbd>Opt</kbd>+<kbd>i</kbd> = multiple item info, one window for the whole selection, stays on top, info changes as selection changes does (_not_ currently supported in fman)
![screen shot 2018-07-31 at 09 53 24](https://user-images.githubusercontent.com/49612/43449135-9d02a1b6-94a7-11e8-9d18-3fb2550d2363.png)

**Comment by mherrmann** [comment]
I agree with @gingerbeardman. The goal of this issue should imho be to implement <kbd>Cmd</kbd>+<kbd>Opt</kbd>+<kbd>i</kbd> in fman similarly to Finder. Unfortunately, updating the info window as the fman selection changes will not be possible.
